__tests__/index.ts

```diff
@@ -19,16 +19,42 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
  */
+
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+
 import { handleRequest } from '../src'
 import { createJsonResponse } from '../src/utils'
 import { mockKV, mockFetch, FetchMock } from './_helper'
 
+const server = setupServer()
+
+beforeAll(() => {
+  server.listen()
+})
+
+afterEach(() => {
+  server.resetHandlers()
+})
+
+afterAll(() => {
+  server.close()
+})
```

Comment:
Another idea. You have noticed that you have copied and pasted those lines to all tests. You can add them globaly in `jest.setup.ts`. Here you need to store the `server` variable in the global scope:

```typescript
global.server = setupServer()

beforeAll(() => {
  global.server.listen()
})

afterEach(() => {
  global.server.resetHandlers()
})

afterAll(() => {
  global.server.close()
})
```

See https://github.com/serlo/api.serlo.org/blob/master/jest.setup.ts for an example.

Note, that you need to add https://github.com/serlo/api.serlo.org/blob/master/jest.setup.ts#L32-L39 as well.